packages/shared/src/components/tooltips/InteractivePopup.tsx

```diff
@@ -25,6 +27,7 @@ interface InteractivePopupProps {
   className?: string;
   position?: InteractivePopupPosition;
   closeOutsideClick?: boolean;
+  alwaysShowClose?: boolean;
```

Comment:
Did we nee do introduce this, does it not always exist?
And would maybe rename it then to `showClose`?

Reply:
For the case of the profile menu it doesn't show all the time for laptop, or should we make it not show when it is profile menu only rather than optional? Wdyt?

Reply:
Since interactive is a generic popup, i'm not sure.
EG how is it handled in squad tour?

Perhaps it's a component inside rather, where we can add the logic?

Reply:
The Squad tour is the old implementation where it uses a modal on mobile. Though on mobile we always have close button, so I guess the better way is to always display it, except Profile menu on desktop? Wdyt?

Reply:
Yeah agreed :)

Reply:
Confirmed to still work 🥳 

![Screenshot 2023-11-14 at 9 57 42 PM](https://github.com/dailydotdev/apps/assets/13744167/8a280e0a-3b72-4de8-ab48-52bfb6f746c1)